crosshatch/highlighter.py

```diff
@@ -0,0 +1,47 @@
+from rich.highlighter import RegexHighlighter
+from rich.theme import Theme
+
+VYXAL_DEFAULT = {
+    "vyxal.number": "bright_green",
+    "vyxal.comment": "green",
+    "vyxal.variable": "bright_blue",
+    "vyxal.string": "red",
+    "vyxal.function": "magenta",
+    "vyxal.function_end": "magenta",
+    "vyxal.constant": "blue",
+    "vyxal.digraph": "bright_cyan",
+    "vyxal.modchar": "cyan",
+    "vyxal.list": "bright_magenta",
+    "vyxal.compressed_number": "yellow",
+    "vyxal.compressed_string": "yellow"
+}
+VYXAL_DEFAULT |= {
+    "vyxal.list_sep": VYXAL_DEFAULT["vyxal.list"],
+    "vyxal.list_end": VYXAL_DEFAULT["vyxal.list"]
+}
```

Comment:
Why not just make these part of the original set and repeat `"bright_magenta"`? Or get rid of `vyxal.list_sep` and `vyxal.list_end` and just `vyxal.list` for separators and list closers.

Reply:
> Why not just make these part of the original set and repeat `"bright_magenta"`? Or get rid of `vyxal.list_sep` and `vyxal.list_end` and just use `vyxal.list` for separators and list closers too.

I'd like to not have duplicates, but unfortunately regex doesn't allow duplicate group names (which are what `rich` uses  for syntax highlighting).